trunk/source/WitnessPrinter/src/de/uni_freiburg/informatik/ultimate/witnessprinter/preferences/PreferenceInitializer.java

```diff
@@ -53,12 +53,14 @@ public enum WitnessVerifierType {
 	public static final String LABEL_GRAPH_DATA_PROGRAMHASH = "Graph data programhash";
 	public static final String LABEL_GRAPH_DATA_ARCHITECTURE = "Graph data architecture";
 	public static final String LABEL_GRAPH_DATA_PRODUCER = "Graph data producer";
+	public static final String LABEL_GRAPH_DATA_PRODUCER_VERSION = "Graph data producer version";

```

Comment:
We can determine our version without the wrapper script hack, why dont we?

Reply:
How can these be done?

Reply:
`UltimateCore.getUltimateVersionString()`